conversions/include/conversions/py_occupancy_grid.py

```diff
@@ -74,6 +74,10 @@ def get_extent(self):
         top_right = self.resolution*(np.flipud(self.shape)) + self.origin
         return self.origin[0], top_right[0], self.origin[1], top_right[1]
 
-    def plot(self):
+    def plot(self, ax):
```

Comment:
For this, can we make the `ax` optional like `ax=None` and then inside do something like `ax = ax if ax else plt.axes()`?
Also, we could passthrough *args, **kwargs, to allow setting other typical imshow arguments (like cmap)

Reply:
yep, I added `ax=None` but I realized that `cmap` is not needed, I don't think **kwargs will be necessary.